### DISCUSSION TITLE
Federated graph scalability

### DISCUSSION THREAD

**Discussion by joshhopkins** [hasura/graphql-engine#8530] [General] [answered]
Are there currently any theoretical limits on the number of remote schemas that can be nested within a federated graph? i.e., n Hasura instances of nested within federated graph.

**Comment by adas98012** [comment, marked as answer]
Hi,
   Thanks for reaching out. Hasura doesn't impose a defacto limit on the number of allowed remote schemas. However,  there may be limits imposed by system resources that you are using. 
   Hope, that answers your question.
